schema/content-descriptor.json

```diff
@@ -23,6 +23,10 @@
         "type": "string",
         "format": "uri"
       }
+    },
+    "annotations": {
+      "id": "https://opencontainers.org/schema/image/descriptor/annotations",
+      "$ref": "defs-image.json#/definitions/annotations"
```

Comment:
oh right. It could be independent. This PR makes an example of it though. So I fixed what I saw.